client/pages/project-edit-modal/modal.template.html.slim

```diff
@@ -17,6 +17,11 @@ form novalidate=true role='form' name='projectForm' ng-submit='projectForm.$vali
       p.help-block
         | The is the name of the project as shown in ProbeDock.
     .form-group
+      label for='projectRepoUrl' Repository URL
+      input#projectRepoUrl.form-control type='string' placeholder='{{ editedProject.repoUrl || "Project repository" }}' maxlength=100 ng-model='editedProject.repoUrl'
```

Comment:
If what you wanted to do was show the original URL in the placeholder when the field is empty, I think that's a bad idea. That would make you think that the URL would remain if you pressed the submit button, when it will actually be removed.